cms/djangoapps/contentstore/signals/handlers.py

```diff
@@ -176,35 +176,38 @@ def listen_for_xblock_published(sender, signal, **kwargs):
     """
     Publish XBLOCK_PUBLISHED signals onto the event bus.
     """
-    get_producer().send(
-        signal=XBLOCK_PUBLISHED, topic='xblock-published',
-        event_key_field='xblock_info.usage_key', event_data={'xblock_info': kwargs['xblock_info']},
-        event_metadata=kwargs['metadata'],
-    )
+    if settings.FEATURES.get("ENABLE_SEND_XBLOCK_EVENTS_OVER_BUS"):
```

Comment:
do we have standard ways of annotating feature flags the same way we do with toggles?  If so we should probably add them here

Reply:
This is the kind of annotation I've seen for feature flags:
```
    # For example in cms/env/common.py
    # .. toggle_name: FEATURES['ENABLE_SEND_XBLOCK_EVENTS_OVER_BUS']
    # .. toggle_implementation: DjangoSetting
    # .. toggle_default: False
    # .. toggle_description: Temporary configuration which enables sending xblock events over the event bus.
    # .. toggle_use_cases: open_edx
    # .. toggle_creation_date: 2023-02-21
    # .. toggle_warning: For consistency in user experience, keep the value in sync with the setting of the same name
    #   in the LMS and CMS.
    # .. toggle_tickets: 'https://github.com/openedx/edx-platform/pull/31813'
    'ENABLE_SEND_XBLOCK_EVENTS_OVER_BUS': False,
```
Shouldn't we also have a default for the feature? As I showed above, in `cms/env/common.py`

Reply:
@mariajgrimaldi @rgraber We could add this to `cms/envs/devstack.py` if not to `common.py` as this is a temporary flag.